srcpkgs/cryfs/template

```diff
@@ -1,22 +1,23 @@
 # Template file for 'cryfs'
 pkgname=cryfs
-version=0.10.2
-revision=5
+version=0.11.2
+revision=1
 create_wrksrc=yes
 build_style=cmake
-configure_args="-DCRYFS_UPDATE_CHECKS=off
- -DBoost_USE_STATIC_LIBS=OFF -DCMAKE_CXX_STANDARD_LIBRARIES=-lfmt"
+configure_args="-DCRYFS_UPDATE_CHECKS=off -DBoost_USE_STATIC_LIBS=OFF 
+ -DCMAKE_CXX_STANDARD_LIBRARIES=-lfmt
```

Comment:
IIRC, `-DCMAKE_CXX_STANDARD_LIBRARIES=-lfmt` isn’t needed here unless you remove `fmt-devel` from `makedepends`.

Reply:
I notice the cryfs template doesn’t have `fmt-devel` in `makedepends` presently. It probably should (see: https://github.com/cryfs/cryfs/issues/432#issuecomment-1183670141).